### ISSUE TITLE
When I run podman, dashboard also shows that OpenShift Local is starting 

### ISSUE THREAD

**Issue by evidolob** [containers/podman-desktop#2365]
### Bug description

When I run podman, dashboard also shows that OpenShift Local is starting:

https://user-images.githubusercontent.com/929743/236393792-3f395df4-4007-4660-a5d5-c29b322e2a7b.mov

And `OpenShift Local` stuck in `starting` phase indefinitely. That is only UI bug, as actual start not performed.

### Operating system

 macOS 13.2.1 Intel

### Version

next (development version)

### Steps to reproduce

Install `OpenShift Local` extension, initialise it, and try to run podman from dashboard.

**Comment by jeffmaury** [comment]
I was able to replicate but can't figure out what's going on.

To reproduce:

1. stop the podman machine
2. Set autostart to false
3. run podman desktop with OpenShift Local extension

On the dashboard page, start Podman. Once Podman is started, OpenShift Local is starting

I tried to add some logs in ProviderConfigured.svelete and seems this is caused by the filed runInProgress being set to true but the only line of code where it is set to true is in the `startProvider` method which is called only when clicking on the Start button.
